docs/source/guides/speed.rst

```diff
@@ -243,15 +276,17 @@ Control Training Epochs
 **Use when:** You run a hyperparameter search to find good initial parameters and want to save time, cost (money), or power (environment).
 It can allow you to be more cost efficient and also run more experiments at the same time.
 
-You can use Trainer flags to force training for a minimum number of epochs or limit to a max number of epochs. Use the `min_epochs` and `max_epochs` Trainer flags to set the number of epochs to run.
+You can use Trainer flags to force training for a minimum number of epochs or limit it to a max number of epochs. Use the ``min_epochs`` and ``max_epochs`` Trainer flags to set the number of epochs to run.
+Setting ``min_epochs=N`` makes sure that the training will run for atleast ``N`` epochs and won't stop until some exception is raised. Setting ``max_epochs=N`` will ensure that training won't happen after
```

Comment:
```suggestion
Setting ``min_epochs=N`` makes sure that the training will run for at least ``N`` epochs and won't stop until some exception is raised. Setting ``max_epochs=N`` will ensure that training won't happen after
```

Reply:
the part about exception is slightly misleading. it's not related to min_epochs at all.

Reply:
updated!